package.json

```diff
@@ -21,8 +21,8 @@
     "socket.io": "^2.4.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
+    "@babel/cli": "^7.15.7",
+    "@babel/preset-env": "^7.15.6",
```

Comment:
Btw do you know if we need babel or not? @sagarchoudhary96  
I'm naive in Node.js 😅

Reply:
babel is a compiler to convert new es6 syntax to compatible in browser. and since we are use that syntax, it will be needed :)

Reply:
Thanks for the answer 😃